int32 pizza_count		# Goal message: number of pizzas for order.
---
string result_message		# Result message: simple string message for the result.
---
string baking_status		# Feedback message: number of pizza is baked.
